src/patternfly/components/Button/styles.scss

```diff
@@ -290,6 +312,71 @@
     }
   }
 
+  // Link and Action Buttons
+  &.pf-m-link,
+  &.pf-m-action {
+    background-color: transparent;
+    box-shadow: none;
+
+    &:hover,
+    &.pf-m-hover,
+    &:active,
+    &.pf-m-active,
+    &:focus,
+    &.pf-m-focus {
+      background-color: transparent;
+      box-shadow: none;
+    }
+  }
+
+  // Link Buttons
+  &.pf-m-link {
+    display: inline-flex;
+    padding: var(--pf-c-button--m-link--PaddingTop) var(--pf-c-button--m-link--PaddingRight) var(--pf-c-button--m-link--PaddingBottom) var(--pf-c-button--m-link--PaddingLeft);
+    color: var(--pf-c-button--m-link--Color);
+
+    > * {
+      margin-right: var(--pf-c-button--m-link--item--MarginRight);
+    }
+
+    &:hover,
+    &.pf-m-hover {
+      color: var(--pf-c-button--m-link--hover--Color);
+    }
+
+    &:active,
+    &.pf-m-active {
+      color: var(--pf-c-button--m-link--active--Color);
+    }
+
+    &:focus,
+    &.pf-m-focus {
+      color: var(--pf-c-button--m-link--focus--Color);
+    }
+  }
+
+  // Action Buttons
+  &.pf-m-action {
+    padding: var(--pf-c-button--m-action--PaddingTop) var(--pf-c-button--m-action--PaddingRight) var(--pf-c-button--m-action--PaddingBottom) var(--pf-c-button--m-action--PaddingLeft);
+    line-height: var(--pf-c-button--m-action--LineHeight);
```

Comment:
why do we change the line height?

Reply:
Assuming that the button should fit the svg within, and this button type would be used exclusively with icons, default line height adds noticeable vertical space.